winbuild/build.rst

```diff
@@ -100,7 +100,7 @@ The following is a simplified version of the script used on AppVeyor::
 
     set PYTHON=C:\Python38\bin
     cd /D C:\Pillow\winbuild
-    C:\Python37\bin\python.exe build_prepare.py -v --depends=C:\pillow-depends
```

Comment:
I would keep this as is to demonstrate that a different Python version can be used to generate the build scripts.